workshop_playbook.md

```diff
@@ -0,0 +1,137 @@
+- List all exercise leads (who consent to being listed) on the website
+- List instructors on the website
+- List expert helpers on the website
+
+### Workshop tools
+
+* create Zoom rooms and communicate their function and addresses
+    * instructor Zoom
+    * exercise Zoom
+* Set up Q&A HackMD (+ archive) with nice URL
+
+
+### Registration
+
+* Create indico event for registration handling
+    * Modify registration confirmation message to include links
+
+### Advertizing
+
+- add all sessions to CodeRefinery calendar separately
+- Create/update advertising texts on the workshop page
+- CodeRefinery Twitter
+- Partner Twitter, retweet and own tweets
+    - Aalto Scientific Computing
+- Partner newsletter
+    - Sigma2
+    - SNIC
+    - ENCCS
+    - CSC
+- Partner websites training calendars
+    - CSC
+    - ENCCS
+    - UiB
+- Partner and other email lists
+    - Aalto STEM students
+    - Aalto triton users
+    - Delta doctoral network
+    - UiB researcher
+    - UiB HPC 
+    - NERSK Bergen
+    - Bjerknes
+    - University of Oslo computational biology 
+    - University of Oslo  Phd and Postdocs 
+    - University of Oslo Dcince contact (?)
+    - Research institutes in all countries
+- Partner posters
+    - Aalto (CS,U,NBE,PHYS,VAARE)
+- Newsletter
+ 
+**Targeted marketing**
+
+Some thoughts on target groups that should be tried to be reached:
+
+- Learners (many sub-categories with different time vs need trade-offs: students, junior researchers, senior researchers, lifelong learning) (join a workshop)
+- Exercise leaders (bring your friends, join to learn more and lead groups)
+- Research leaders (people who can tell their students they need to attend CR)
+- University staff (Computing, open sci, etc.) (people who can serve as local organizers and serve as local helpers)
+- Potential instructors (teach, etc.)
+- High-level management (provide us funding)
+
+### Communication 
+
+- Draft email templates early; collaborate
+  - Observers (+ EL onboarding info)
+  - unsure registrations
+  - Exercise leads (+ onboarding info)
+  - Stream learners
+  - update indico auto-reply for last minute registrations
+  - remind team contacts to update team info
+- Calendar invites to EL/EH/instructors/organization team
+- Exercise leader onboarding summary email
+
+## Continuous / every few days:
+- check indico and follow up on registrations
+   - look at team registrations and check for problems
+     - identify where we need to do something. e.g. persons who wanted an in-person room. or persons who were not sure but wanted to be informed. basically, "playing" with the registration data now is important to not forget anybody and not to forget follow-up
+   - look at exercise leads and make sure they registered also for the onboarding
+   - try to approve as early as possible
+ - check Freshdesk for new support requests
+
+## During the workshop
+- after every workshop day, send summary and preparation for next day email to all
```

Comment:
thought: I think we should do more updating of the news on the website.  this can be done even during the day as the speakers mention things for the next day.
```suggestion
- during/after each workshop day, collect relevant news on the workshop webpage
- after every workshop day, send summary and preparation for next day email to all (can be based on the website news and link there so that it can be updated)
```